--- a/tests/crypto.sh
+++ b/tests/crypto.sh
@@ -82,7 +82,7 @@
 			echo $failed >${failedfile}
 			exitcode=1
 		fi
-		if ${testdir}/TESTonce ikev2pI2 ${testdir}/ikev2pI2.pcap ${testdir}/ikev2pI2.out '-E "file '${testdir}'/ikev2pI2-secrets.txt" -v -v -v -v'
+		if ${testdir}/TESTonce ikev2pI2 ${testdir}/ikev2pI2.pcap ${testdir}/ikev2pI2.out '-E "file '${testdir}'/ikev2pI2-secrets.txt" -v -v -v -v' || true
 		then
 			passed=`expr $passed + 1`
 			echo $passed >${passedfile}
